packages/components/src/search-list-control/style.scss

```diff
@@ -202,11 +206,7 @@
 	}
 }
 
-.components-panel {
-	.woocommerce-search-list {
-		padding: 0;
```

Comment:
This will fix the visual glitch in the screenshot I posted in the PR description: there was no space between the list and the label below:

![imatge](https://user-images.githubusercontent.com/3616980/102517770-bd47ce00-4090-11eb-8f6b-950c2f652938.png)